mflowgen/Tile_PE/construct.py

```diff
@@ -264,14 +264,24 @@ def construct():
   if pwr_aware:
      init.update_params( { 'flatten_effort': parameters['flatten_effort'] }, True ) 
      pwr_aware_gls.update_params( { 'design_name': parameters['design_name'] }, True ) 
-  
+     
+     init.extend_postconditions(         ["assert 'Clamping logic structure in the SBs and CBs is not maintained' in File( 'mflowgen-run.log' )"] )  
```

Comment:
Same as for Tile_MemCore.

Reply:
Ankita changed this herself in future commit